treeherder/model/sql/jobs.json

```diff
@@ -349,7 +349,7 @@
 
             "sql":"SELECT s.`signature`,
                     GROUP_CONCAT(
-                        CONCAT_WS('=', s.`property`, s.`value`) SEPARATOR ',')
+                        CONCAT_WS('=', s.`property`, s.`value`) SEPARATOR '\n')
```

Comment:
Because the series signature list could potentially have "," in it if there is more than one series, in which case we can't split on ','. So we need to use some other kind of delimiter. '\n' seemed like the obvious one. :)